src/main/scala/Main.scala

```diff
@@ -1,8 +1,84 @@
+
+
+import newmodel.Package
+import newmodel._
+
+import scala.collection.mutable.ListBuffer
+
 /**
  * Created by yaroslav on 11.05.2015.
  */
 object Main {
-  def main(args:Array[String]) ={
-    println("hello world")
+
+  object DummyIndex extends Index{
+    val docElements = ListBuffer.empty[DocElement]
+    override def getConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getKnownSubclasses(id: ID): Seq[ID] = ???
+
+    override def getShadowedConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getLinearSupertypesBy(id: ID): Seq[ID] = ???
+
+    override def getCompanionFor(id: ID): ID = ???
+
+    override def getDocElementBy(id: ID): DocElement = docElements.find((x) => x.id == id).get
```

Comment:
I would like us to find the sweet spot between not spending time on doing something, that meta will do better in one year (i.e. building a better Scala AST), but also not depending on something, that won't be finished or heavily changing during this project (meta's better ASTs). Working directly on Scaladoc's model however means that @kolowheel will have to work with the Scala compiler source code, which is quite resource demanding on his machine. And it would mean that the work doesn't easily transfer to meta/dotty. Thoughts anyone?